src/Implement/ResponseHandler.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <iostream>
+#include <sys/socket.h>
+
+#include "OperateData.h"
+#include "ResponseHandler.h"
+#include "ResponseEncoder.h"
+
+void ResponseHandler::Send(const std::string &for_client_fd, const std::unique_ptr<OperateData> &data) {
+    
+    try {
+        ResponseEncoder *tmp_encoder = ResponseEncoder::GetInstance();
+        // Encode header and body to OperateData
+        std::string response = tmp_encoder->Encode(data);
```

Comment:
Please leave things that doesn't trigger exception due to wrong client fd type out of the try-catch block.

Reply:
Done!